README.md

```diff
@@ -39,8 +51,13 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
     console.log("success!");
   });
 ```
+
 ### Question 6
-Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`. 
+Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`.
+
+  $('#submit').on('click', UserApp.submit_user);
```

Comment:
We already talked about this in class, but just to reinforce, the answer here is `var user_input = $('#color').val()`